package.json

```diff
@@ -13,6 +13,7 @@
     "package-term": "pnpm --filter=@gravitational/teleterm package",
     "storybook": "./web/scripts/run-storybook.sh",
     "storybook-smoke-test": "storybook dev -p 9002 -c web/.storybook --ci --smoke-test",
+    "test-storybook": "NODE_TLS_REJECT_UNAUTHORIZED=0 test-storybook -c web/.storybook --url https://localhost:9002 --skipTags=skip-test",
```

Comment:
`NODE_TLS_REJECT_UNAUTHORIZED=0` is likely not going to be needed in the final version, assuming we want to run the test runner against non-dev version of storybook.

Reply:
@ravicious Skipping this parameter causes the test runner to simply claim that there's no Storybook running. It refuses to uses a site with self-signed certificates. Does the non-dev version use proper certs?

Reply:
>Skipping this variable causes the test runner to simply claim that there's no Storybook running. It refuses to uses a site with self-signed certificates.

I'm wondering why is this happening, playwright's chromium should trust the certs generated with `mkcert`, no? It works for me without that env var.

Reply:
The non-dev version from https://github.com/gravitational/teleport/pull/45116#discussion_r1705617377 is served over HTTP. The only reason we serve Storybook over HTTPS in the dev version is because of HSTS set by Teleport if you access Teleport over `localhost` (see https://github.com/gravitational/teleport/issues/32256).

It works for me without the env var too because the certs I have in `web/certs` are trusted by my system. I generated them with mkcert and `mkcert --install` adds its CA to the system trust store.

Reply:
@ravicious I don't know... I did everything according to the procedure outlined in `web/README.md`. Ran `mkcert --install`, generated the certificate files... I even regenerated them right now, but though the browser itself recognizes the certificate, `pnpm test-storybook` fails.